activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -79,6 +79,8 @@ def aliased_types(name, fallback)
 
     AddColumnDefinition = Struct.new(:column) # :nodoc:
 
+    RemoveColumnDefinition = Struct.new(:name) # :nodoc:
```

Comment:
Shouldn't the attribute be called `column` to be consistent with `AddColumnDefinition`?